asciidoctorj-core/src/test/java/org/asciidoctor/WhenAsciiDocIsRenderedToDocument.java

```diff
@@ -298,20 +298,20 @@ public void should_get_attributes() {
             "\n";
 
         Document document = asciidoctor.load(documentWithAttributes, new HashMap<String, Object>());
-        List<StructuralNode> blocks = document.getBlocks();
+        List<? extends StructuralNode> blocks = document.getBlocks();
```

Comment:
Please don't do this.
I tend to show these tests to answer questions.
`List<? extends StructuralNode>` hides the possibility that this list can be mutated.
But we encourage that, see for example here:
https://github.com/asciidoctor/asciidoctorj/blob/5b8018792c787f59df8ee099d5f8393f68e7591b/asciidoctorj-core/src/test/java/org/asciidoctor/extension/TerminalCommandTreeprocessor.java#L31-L39

Reply:
Of course. Sorry I missed that one by reverting...

Reply:
Ah ok, no worries.
I thought this was intentional :-)